File: Abhash Goyal/Arrays/1637. Widest Vertical Area Between Two Points Containing No Points.cpp
class Solution {
public:
    int maxWidthOfVerticalArea(vector<vector<int>>& points) {
          int maxdis=0;
          sort(points.begin(),points.end());

        for (int i = 1 ; i < points.size();i++){

            maxdis =  max(maxdis, points[i][0] - points[i-1][0]);
        }
        return maxdis;
        
        
    }
};

File: Vishakha Mishra /ARRAY/1637. Widest Vertical Area Between Two Points Containing No Points.java
class Solution {
    public int maxWidthOfVerticalArea(int[][] points) {
        
        int n = points.length;
        int[] arr = new int[n];
        
        int temp=0;
        for (int i = 0; i < n; i++) {
            arr[i] = points[i][0];
        }
        
        Arrays.sort(arr);
        
        for(int i=0;i<n-1;i++){
         int v = arr[i+1]-arr[i];
           if(v > temp){
            temp=v;                                                                              
        }
        }
        return temp;
    }
}
